### PR TITLE
Kmh/feature branch example

### PR DESCRIPTION
**Pull request by kristyh32**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ Person A should modify content here.
...
+Modifyin' up in here. 
 
 ## Person C:
 
 Person C should modify content here.
 
 ## Person D:
 
-Person D should modify content here.
+We ALL have rumps.
```
Comment (juliabouv) [inline README.md:19]: It all makes sense now!

**Review by juliabouv — COMMENTED** [review]
Wooooow!

**Review by juliabouv — CHANGES_REQUESTED** [review]
Nope.

**Review by juliabouv — CHANGES_REQUESTED** [review]
Specific nope

**Comment by kristyh32** [comment]
Be better Julia

**Pull request merged**